cmake/CMakeLists.txt

```diff
@@ -36,35 +36,50 @@ endif()
-option(COLLADA_SUPPORT "Build IfcConvert with COLLADA support (requires OpenCOLLADA)." ON)
-option(IFCXML_SUPPORT "Build IfcParse with ifcXML support (requires libxml2)." ON)
+option(WASM_BUILD "Build a WebAssembly binary." OFF)
+
 option(ENABLE_BUILD_OPTIMIZATIONS "Enable certain compiler and linker optimizations on RelWithDebInfo and Release builds." OFF)
-option(BUILD_PACKAGE "" OFF)
+option(BUILD_SHARED_LIBS "Build IfcParse and IfcGeom as shared libs (SO/DLL)." OFF)
+option(MSVC_PARALLEL_BUILD "Multi-threaded compilation in Microsoft Visual Studio (/MP)" OFF)
+option(USE_VLD "Use Visual Leak Detector for debugging memory leaks, MSVC-only." OFF)
+option(USE_MMAP "Adds a command line options to parse IFC files from memory mapped files using Boost.Iostreams" OFF)
+option(NO_WARN "Disable all warnings" OFF)
+
 option(BUILD_IFCGEOM "Build IfcGeom." ON)
-option(BUILD_DOCUMENTATION "Build IfcOpenShell Documentation." OFF)
 option(BUILD_IFCPYTHON "Build IfcPython." ON)
+option(BUILD_CONVERT "Build IfcConvert executable." ON)
+option(BUILD_DOCUMENTATION "Build IfcOpenShell Documentation." OFF)
 option(BUILD_EXAMPLES "Build example applications." ON)
 option(BUILD_GEOMSERVER "Build IfcGeomServer executable." ON)
-option(BUILD_CONVERT "Build IfcConvert executable." ON)
+option(BUILD_IFCMAX "Build IfcMax, a 3ds Max plug-in, Windows-only." OFF)
+option(BUILD_QTVIEWER "Build IfcOpenShell Qt GUI Viewer" OFF) # QtViewer requires Qt6
+option(BUILD_PACKAGE "" OFF)
+
+option(COLLADA_SUPPORT "Build IfcConvert with COLLADA support (requires OpenCOLLADA)." ON)
+option(GLTF_SUPPORT "Build IfcConvert with glTF support (requires json.hpp)." OFF)
 option(HDF5_SUPPORT "Enable HDF5 support (requires HDF5, zlib)" ON)
-option(USE_VLD "Use Visual Leak Detector for debugging memory leaks, MSVC-only." OFF)
-option(USE_MMAP "Adds a command line options to parse IFC files from memory mapped files using Boost.Iostreams" OFF)
-option(BUILD_SHARED_LIBS "Build IfcParse and IfcGeom as shared libs (SO/DLL)." OFF)
-option(MSVC_PARALLEL_BUILD "Multi-threaded compilation in Microsoft Visual Studio (/MP)" OFF)
+option(IFCXML_SUPPORT "Build IfcParse with ifcXML support (requires libxml2)." ON)
+option(USD_SUPPORT "Build IfcConvert with USD support (requires pixar's USD library)." OFF)
 
-option(NO_WARN "Disable all warnings" OFF)
-option(WASM_BUILD OFF)
+option(USERSPACE_PYTHON_PREFIX "Installs IfcPython for the current user only instead of system-wide." OFF)
+option(ADD_COMMIT_SHA "Add commit sha and branch in version number, warning results in many rebuilds, requires git" OFF)
 
-if(${BUILD_CONVERT})
-    option(GLTF_SUPPORT "Build IfcConvert with glTF support (requires json.hpp)." OFF)
-    option(USD_SUPPORT "Build IfcConvert with USD support (requires pixar's USD library)." OFF)
+if(MINIMAL_BUILD)
+    message(STATUS "Setting options for minimal build")
+    set(BUILD_CONVERT OFF)
```

Comment:
I think MINIMAL_BUILD did not exclude building IfcConvert.

Reply:
Yes, you are right. It check that BUILD_CONVERT was set if GLTF_SUPPORT was set in line 173. Will delete this line.